### PR TITLE
Add support for NuGet Readme

### PR DESCRIPTION
**Pull request by campersau**
Fixes https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/issues/1095

### CODE REVIEW

File: Directory.Build.targets [hunk 1]
```diff
@@ -4,10 +4,6 @@
     <UseWpf>true</UseWpf>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
-
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
```
Comment (campersau) [inline Directory.Build.targets:8]: @clairernovotny was this a special build related to some signature stuff?
I have removed the `netcoreapp3.1` target for now.

Comment (clairernovotny) [reply Directory.Build.targets:8]: We need the .NET Core App 3.1 version as other things are consuming the Core nuget package for symbol validation. The Core version is using a private build of NuGet that does the signature validation.

**Review by clairernovotny — CHANGES_REQUESTED** [review]
Looks great other than the .NET Core 3.1 removal. If you could please add that back, I can merge.

**Comment by clairernovotny** [comment]
Could you also please bump the `version.json` to be 5.10? I've been keeping the minor in sync with NuGet to get a sense of what features are where.

**Comment by campersau** [comment]
@clairernovotny I have bumped the version to `5.10`.

**Pull request merged**